db/seeds.d/106-mail_notifications.rb

```diff
@@ -31,6 +31,13 @@
      :mailer => 'Katello::ErrataMailer',
      :method => 'promote_errata',
      :subscription_type => 'alert'
+    },
+
+    {:name => :subscription_expiry,
+     :description => N_('A list of Subscriptions expiring in 30 days'),
```

Comment:
```suggestion
     :description => N_('A list of subscriptions expiring within the next 30 days'),
```
This is a bit more clear (also, no need to capitalize "subscriptions")